datacenter/ucp/2.1/guides/admin/configure/add-sans-to-cluster.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Add SANs to cluster certificates
+description: Learn how to add new SANs to cluster nodes, allowing you to connect to UCP with a different hostname
+keywords: Docker, cluster, nodes, labels, certificates, SANs
+---
+
+When you connect to UCP, you need to make sure that the hostname that you use to connect is recognized
```

Comment:
Updated the intro.

If you use your own TLS certs, you actually can add SANs as much as you want; nothing will break. We will happily add the new SANs to those certs.